data/tribes/scripting/starting_conditions/barbarians/poor_hamlet.lua

```diff
@@ -0,0 +1,78 @@
+-- =======================================================================
+--                 Minimum Starting conditions for Barbarians
+-- =======================================================================
+
+include "scripting/infrastructure.lua"
+
+set_textdomain("tribes")
+
+return {
+   -- TRANSLATORS: This is the name of a starting condition
+   descname = _ "Poor Hamlet",
+   -- TRANSLATORS: This is the tooltip for the "Poor Hamlet" (minimum) starting condition
+   tooltip = _"Start the game with just a warehouse and the bare minimum for bootstrapping an economy. Warning: the AI can't successfully start from this",
+   func = function(player, shared_in_start)
+      local sf = wl.Game().map.player_slots[player.number].starting_field
+      if shared_in_start then
+         sf = shared_in_start
+      else
+         player:allow_workers("all")
+      end
+
+      prefilled_buildings(player, { "barbarians_warehouse", sf.x, sf.y,
+         wares = {
+            log = 3,
+            coal = 1,
+            hammer = 2,
+            felling_ax = 1,
+            pick = 1,
+            --Forester, gardener:
```

Comment:
Comments should always start with a whitespace after the `--`

Reply:
ok sorry